packages/daemon/src/daemon.js

```diff
@@ -733,17 +605,7 @@ export const main = async (powers, locator, pid, cancel, cancelled) => {
     },
   );
 
-  const pidPath = powers.joinPath(locator.ephemeralStatePath, 'endo.pid');
-
-  await powers
-    .readFileText(pidPath)
-    .then(pidText => {
-      const oldPid = Number(pidText);
-      powers.kill(oldPid);
-    })
-    .catch(() => {});
-
-  await powers.writeFileText(pidPath, `${pid}\n`);
+  await powers.updateDaemonPid(locator, pid);
```

Comment:
i originally called this `finalizeInitialization` as a sort of generic hook into the platform. might make more sense than this specific name